newrelic/config.py

```diff
@@ -189,10 +189,15 @@ def _map_console_listener_socket(s):
 
 
 def _merge_ignore_status_codes(s):
-    return newrelic.core.config._parse_ignore_status_codes(
+    return newrelic.core.config._parse_error_status_codes(
```

Comment:
If we're changing the name of this, can we just make it `_parse_status_codes`? There's really nothing error specific about it. The name bothers me a bit.